core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -60,7 +70,20 @@ public void createTable(
   @Override
   public void createNamespace(String namespace, Map<String, String> options)
       throws ExecutionException {
-    CreateKeyspace query = SchemaBuilder.createKeyspace(quoteIfNecessary(namespace));
+    try {
+      createKeyspace(namespace, options);
+      createMetadataKeyspaceIfNotExists();
```

Comment:
Just a question. 
I think we haven't managed a metadata keyspace for Cassandra since Cassandra manages keyspace metadata by itself, but now are we going to do it for some reason?
If so, what is the reason?

Reply:
The goal is to be able to manage keyspaces that are only created through Scalar DB. We don't think we should be returning keyspaces that are not Scalar DB compatible with `admin.getNamespaceNames()`. 
Since with the built-in Cassandra keyspace metadata manager, we can't know if a keyspace/table was created through Scalar DB, so adding a table to manage Scalar DB keyspace was necessary.  

Similarly, we are also thinking of making `admin.getNamespaceTableNames()` only return tables created through Scalar DB, which implies we will need to create a table to manage table metadata for Cassandra as it is done on other admin implementations.

Reply:
OK, that sounds good to me.
So, it's a bit off-topic, but I'm wondering how Scalar DB would work if it uses JDBC native transactions.
Do we also manage metadata even if it uses JDBC native transactions?
(cc: @brfrn169)

Reply:
For JDBC native transactions, table and namespace metadata are also managed, via the [JdbcTransactionAdmin](https://github.com/scalar-labs/scalardb/blob/master/core/src/main/java/com/scalar/db/transaction/jdbc/JdbcTransactionAdmin.java) 

As with the other usage mode (consensus-commit or storage), the table metadata are used to validate if the given operation is valid (check wrong keys, check wrong projected columns names, etc.).

Reply:
Oh, OK. Thank you!